addons/web/static/src/views/form/form_controller.scss

```diff
@@ -240,9 +240,13 @@
     // No sheet
     &.o_form_nosheet, .o_form_nosheet {
         @include o-webclient-padding($top: $o-sheet-vpadding, $bottom: $o-sheet-vpadding);
+        background-color: var(--formView-sheet-background);
 
         .o_form_statusbar {
             margin: (-$o-sheet-vpadding) (-$o-horizontal-padding) $o-sheet-vpadding (-$o-horizontal-padding);
+            background-color: map-get($grays, '100');
```

Comment:
The rules' order in incorrect (should be from the "outside" to the "inside") ;)
https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#properties-order